### PR TITLE
Code Quality Fixes

### PR DESCRIPTION
**Pull request by lakshyacis**
CODE QUALITY FIXES
socks4a pylint fixes,
socks5 flake8 fixes,
stats flake8 fixes,
stats pylint fixes,
tls flake8 fixes,
tls pylint fixes,
udp pylint fixes,
networkthread pylint fixes,
node pylint fixes,
proxy pylint fixes.

### CODE REVIEW

File: src/network/socks4a.py [hunk 1]
```diff
@@ -82,7 +87,7 @@ def state_auth_done(self):
             self.append_write_buf(self.ipaddr)
         except socket.error:
             # Well it's not an IP number,  so it's probably a DNS name.
-            if Proxy._remote_dns:
+            if Proxy._remote_dns:       # pylint: disable=protected-access
```
Comment (g1itch) [inline src/network/socks4a.py]: `self._remote_dns`

**Comment by g1itch** [comment]
What a problem with flake8 in `network.socks5`?

```sh
$ flake8 -v src/network/socks5.py
flake8.plugins.manager    MainProcess     86 INFO     Loading entry-points for "flake8.extension".
flake8.plugins.manager    MainProcess    111 INFO     Loading entry-points for "flake8.listen".
flake8.plugins.manager    MainProcess    111 INFO     Loading entry-points for "flake8.report".
flake8.plugins.manager    MainProcess    112 INFO     Loading plugin "C90" from entry-point.
flake8.plugins.manager    MainProcess    117 INFO     Loading plugin "pycodestyle.comparison_type" from entry-point.
flake8.plugins.manager    MainProcess    124 INFO     Loading plugin "pycodestyle.missing_whitespace_after_import_keyword" from entry-point.
flake8.plugins.manager    MainProcess    124 INFO     Loading plugin "pycodestyle.imports_on_separate_lines" from entry-point.
flake8.plugins.manager    MainProcess    124 INFO     Loading plugin "pycodestyle.continued_indentation" from entry-point.
flake8.plugins.manager    MainProcess    125 INFO     Loading plugin "pycodestyle.maximum_line_length" from entry-point.
flake8.plugins.manager    MainProcess    125 INFO     Loading plugin "pycodestyle.bare_except" from entry-point.
flake8.plugins.manager    MainProcess    125 INFO     Loading plugin "pycodestyle.ambiguous_identifier" from entry-point.
flake8.plugins.manager    MainProcess    125 INFO     Loading plugin "pycodestyle.python_3000_not_equal" from entry-point.
flake8.plugins.manager    MainProcess    125 INFO     Loading plugin "pycodestyle.trailing_whitespace" from entry-point.
flake8.plugins.manager    MainProcess    125 INFO     Loading plugin "F" from entry-point.
flake8.plugins.manager    MainProcess    136 INFO     Loading plugin "pycodestyle.whitespace_around_operator" from entry-point.
flake8.plugins.manager    MainProcess    136 INFO     Loading plugin "pycodestyle.python_3000_raise_comma" from entry-point.
flake8.plugins.manager    MainProcess    136 INFO     Loading plugin "pycodestyle.comparison_negative" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.python_3000_has_key" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.blank_lines" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.python_3000_backticks" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.indentation" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.tabs_or_spaces" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.module_imports_on_top_of_file" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.missing_whitespace" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.whitespace_around_named_parameter_equals" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.explicit_line_join" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.whitespace_around_comma" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.whitespace_before_parameters" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.missing_whitespace_around_operator" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.compound_statements" from entry-point.
flake8.plugins.manager    MainProcess    137 INFO     Loading plugin "pycodestyle.trailing_blank_lines" from entry-point.
flake8.plugins.manager    MainProcess    138 INFO     Loading plugin "pycodestyle.extraneous_whitespace" from entry-point.
flake8.plugins.manager    MainProcess    138 INFO     Loading plugin "pycodestyle.whitespace_around_keywords" from entry-point.
flake8.plugins.manager    MainProcess    138 INFO     Loading plugin "pycodestyle.tabs_obsolete" from entry-point.
flake8.plugins.manager    MainProcess    138 INFO     Loading plugin "pycodestyle.break_around_binary_operator" from entry-point.
flake8.plugins.manager    MainProcess    138 INFO     Loading plugin "pycodestyle.whitespace_before_comment" from entry-point.
flake8.plugins.manager    MainProcess    138 INFO     Loading plugin "pycodestyle.comparison_to_singleton" from entry-point.
flake8.plugins.manager    MainProcess    138 INFO     Loading plugin "default" from entry-point.
flake8.plugins.manager    MainProcess    139 INFO     Loading plugin "quiet-filename" from entry-point.
flake8.plugins.manager    MainProcess    139 INFO     Loading plugin "pylint" from entry-point.
flake8.plugins.manager    MainProcess    139 INFO     Loading plugin "quiet-nothing" from entry-point.
flake8.checker            MainProcess    153 INFO     Making checkers
flake8.checker            MainProcess    159 INFO     Checking 1 files
flake8.main.application   MainProcess    351 INFO     Finished running
flake8.main.application   MainProcess    352 INFO     Reporting errors
flake8.main.application   MainProcess    353 INFO     Found a total of 0 violations and reported 0
```

**Found a total of 0 violations and reported 0**

**Comment by lakshyacis** [comment]
W504 line break after binary operator

**Comment by g1itch** [comment]
Hate it 8( At first it warned about line break **before** binary operator (some versions keep warning). And now it turns out that the right way is to break before: https://bugs.python.org/issue26763

https://github.com/PyCQA/pycodestyle/issues/498

**Comment by lakshyacis** [comment]
Okay, will try to ignore that warning from next time.

**Comment by lakshyacis** [comment]
> Hate it 8( At first it warned about line break **before** binary operator (some versions keep warning). And now it turns out that the right way is to break before: https://bugs.python.org/issue26763
> 
> [PyCQA/pycodestyle#498](https://github.com/PyCQA/pycodestyle/issues/498)

File: src/network/udp.py [hunk 2]
```diff
@@ -124,7 +129,7 @@ def handle_read(self):
 
         self.destination = state.Peer(*addr)
         encodedAddr = protocol.encodeHost(addr[0])
-        if protocol.checkIPAddress(encodedAddr, True):
+        if protocol.checkIPAddress(encodedAddr, True):      # pylint: disable=simplifiable-if-statement
```
[the remark this answers is not in the document]
Comment (lakshyacis) [reply src/network/udp.py]: okay.

**Pull request merged**